morango/models.py

```diff
@@ -351,7 +351,8 @@ def calculate_filter_max_counters(cls, filters):
 
         with connection.cursor() as cursor:
             cursor.execute(filter_max_calculation)
-            return dict(cursor.fetchall())
+            # try to get hex value because postgres returns values as uuid
```

Comment:
yeah this issue came out because of testing in postgres.

Reply:
Right -- but do you think we need to worry about this situation more broadly in Kolibri, beyond Morango, with Postgres use?

Reply:
I want to say no because things probably would've been uncovered with the extensive tests within kolibri. I think django would handle that mechanism pretty well but here we are doing raw sql stuff.